lib/core/src/client/preview/start.js

```diff
@@ -316,17 +316,25 @@ export default function start(render, { decorateStory } = {}) {
       reqs = [loadable];
     }
 
-    let currentExports;
+    let currentExports = new Set();
     if (reqs) {
-      currentExports = new Set();
       reqs.forEach(req => {
         req.keys().forEach(filename => {
           const fileExports = req(filename);
           currentExports.add(fileExports);
         });
       });
     } else {
-      currentExports = new Set(loadable());
+      const exported = loadable();
+      if (Array.isArray(exported) && !exported.find(obj => !obj.default)) {
+        currentExports = new Set(exported);
+      } else if (exported) {
+        logger.warn(
+          `Loader function passed to 'configure' should return void or an array of module exports. Received: ${JSON.stringify(
```

Comment:
If they forget to `export default` in a single CSF file they will hit this message. Maybe add something about that too?

Reply:
So, is the `export default` kind of mandatory? I know some people advocating `export default` could be harder for refactoring and some team even use ESLint to ban default export.  Since it is possible to export multiple stories using named export.  Maybe that is what really need to be addressed.

Reply:
Export default is the CSF equivalent to `storiesOf('component-name', module)` so yes it's mandatory.